src/main/java/seedu/tp/flashcard/FlashcardList.java

```diff
@@ -21,9 +21,11 @@ public FlashcardList() {
      * Adds a flashcard to the list.
      *
      * @param flashcard the task to be added to the list
+     * @return
      */
-    public void addFlashcard(Flashcard flashcard) {
+    public FlashcardList addFlashcard(Flashcard flashcard) {
```

Comment:
There is no need to return the `FlashcardList` as we are just making modifications on the object itself and the caller must have the reference to the object

Reply:
Actually, returning FlashcardList makes it easier to do sequential additions. 
For  example (this is a constant I use in the JUnit test),
```
public static final FlashcardList FLASHCARD_LIST = new FlashcardList()
            .addFlashcard(new EventFlashcard(FLASHCARD_NAME, START_LOCAL_DATE, END_LOCAL_DATE, SUMMARY, DETAILS))
            .addFlashcard(new PersonFlashcard(FLASHCARD_NAME, START_LOCAL_DATE, END_LOCAL_DATE, SUMMARY, DETAILS))
            .addFlashcard(new OtherFlashcard(FLASHCARD_NAME, SUMMARY, DETAILS));
```

Reply:
Ohhh I see. Understood!